src/parser/meta.rs

```diff
@@ -598,7 +600,7 @@ mod tests {
         let input = b"HD\r\n";
         let (remaining, response) = parse_meta_get_data_value(input).unwrap();
 
-        assert_eq!(remaining, b"\r\n");
+        assert_eq!(remaining, b"");
```

Comment:
Tests now confirm that the buffer is left empty instead of having a remaining `crlf` at the end of parsing.